Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackParamsEstimationAlgorithm.hpp

```diff
@@ -71,9 +71,9 @@ class TrackParamsEstimationAlgorithm final : public IAlgorithm {
     double bFieldMin = 0.1 * Acts::UnitConstants::T;
     /// Initial covariance matrix diagonal.
     std::array<double, 6> initialSigmas = {
-        25 * Acts::UnitConstants::um,       100 * Acts::UnitConstants::um,
-        0.02 * Acts::UnitConstants::degree, 0.02 * Acts::UnitConstants::degree,
-        0.1 / Acts::UnitConstants::GeV,     10 * Acts::UnitConstants::ns};
+        1 * Acts::UnitConstants::mm,     1 * Acts::UnitConstants::mm,
+        1 * Acts::UnitConstants::degree, 1 * Acts::UnitConstants::degree,
+        0.1 / Acts::UnitConstants::GeV,  1 * Acts::UnitConstants::ns};
```

Comment:
is there a reason you decrease the standard deviation of time while you increase the standard deviation of the impact parameters & angles?

Reply:
the problem is that 1ns is a very big numerical value compared to 0.02" or um. I just push them up without diverging too much